modules/dnn/perf/perf_net.cpp

```diff
@@ -111,6 +110,7 @@ PERF_TEST_P_(DNNTestNetwork, Inception_5h)
 
 PERF_TEST_P_(DNNTestNetwork, SSD)
 {
+    applyTestTag(CV_TEST_TAG_DEBUG_VERYLONG);
```

Comment:
empty line after that https://github.com/opencv/opencv/blob/4.9.0/modules/dnn/perf/perf_net.cpp#L144

to avoid merge conflicts in the future: